raiden/transfer/mediated_transfer/mediator.py

```diff
@@ -855,8 +855,10 @@ def events_to_remove_expired_locks(
         secrethash = mediator_state.secrethash
         lock = None
         if secrethash in channel_state.our_state.secrethashes_to_lockedlocks:
+            assert secrethash not in channel_state.our_state.secrethashes_to_unlockedlocks
             lock = channel_state.our_state.secrethashes_to_lockedlocks.get(secrethash)
         elif secrethash in channel_state.our_state.secrethashes_to_unlockedlocks:
+            assert secrethash not in channel_state.our_state.secrethashes_to_lockedlocks
```

Comment:
@rakanalh but you are right that the second assert it not needed due toi the `if` `elif`. Just keep the first one only.